nextflow.config

```diff
@@ -23,34 +23,104 @@ params {
     // Main options
     analysis_type              = 'wgs'
     bait_padding               = 100
+    run_rtgvcfeval             = false
     save_mapped_as_cram        = false
-    skip_cnv_calling           = false
+    skip_eklipse               = false
+    skip_fastp                 = false
+    skip_fastqc                = false
+    skip_gens                  = true
+    skip_germlinecnvcaller     = false
+    skip_haplocheck            = false
+    skip_peddy                 = false
+    skip_me_annotation         = false
+    skip_mt_annotation         = false
+    skip_qualimap              = false
     skip_snv_annotation        = false
     skip_sv_annotation         = false
-    skip_mt_analysis           = false
-    gens_switch                = false
+    skip_me_annotation         = false
+    skip_mt_annotation         = false
+    skip_mt_subsample          = false
+    skip_vcf2cytosure          = true
+    skip_vep_filter            = false
     cadd_resources             = null
     platform                   = 'illumina'
 
+    // Bam_qc
+    ngsbits_samplegender_method = 'xy'
+
+    // File params
+    fasta                           = null
```

Comment:
Apparently, that causes some problems. Read about it here:

https://nfcore.slack.com/archives/C04QR0T3G3H/p1709811356971549

Reply:
Fixed here https://github.com/nf-core/raredisease/pull/530